frontend/.env

```diff
@@ -1 +1,3 @@
-SKIP_PREFLIGHT_CHECK=true
\ No newline at end of file
+SKIP_PREFLIGHT_CHECK=true
+
+REACT_APP_API_URL=http://localhost:5000
```

Comment:
I think better would be setting
`REACT_APP_API_URL=https://coders-camp-organization-app.herokuapp.com`

Reply:
I changed url 👍